src/components/Planner/PlannerTimeline/PlannerTimeline.jsx

```diff
@@ -48,8 +56,16 @@ const PlannerTimeline = () => {
 
   useEffect(() => {
     updateTimeline();
+    getDayData();
   }, []);
 
+  useEffect(() => {
+    if (dataShouldRevalidate) {
+      getDayData();
+      setShouldDataRevalidate(false);
+    }
+  }, [dataShouldRevalidate])
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'getDayData'. Either include it or remove the dependency array.